src/chain/chain.rs

```diff
@@ -835,6 +847,15 @@ impl Chain {
         elders
     }
 
+    // Compare candidates for the next elders. The one comparing `Less` is more likely to become
+    // elder.
+    fn cmp_elder_candidates(&self, lhs: &MemberInfo, rhs: &MemberInfo) -> Ordering {
+        // Older nodes are preferred. In case of a tie, nodes joining earlier are preferred.
+        rhs.age_counter
+            .cmp(&lhs.age_counter)
+            .then(lhs.section_version.cmp(&rhs.section_version))
```

Comment:
Yeah, I remember us discussing this at some point, just not sure what it was exactly anymore. So chance is this will become useful for other stuff too.